app/forms/keywords_form.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class KeywordsForm
```

Comment:
@aetheryna How about following this [convention](https://nimblehq.co/compass/development/code-conventions/ruby/ruby-on-rails/#convention-38)?
So here, we will have the `initialize` method assigning `user` and so on.
And then calling `.save`, we don't have to send user again

Reply:
Good idea! Made the change here https://github.com/aetheryna/google-scraper-ruby/pull/34/commits/40f76b1ebffdddf0f3a49ba6f8c7d370bf2afcad 😄